appengine/bigquery/requirements.txt

```diff
@@ -1 +1 @@
-google-api-python-client
+google-api-python-client==1.4.2
```

Comment:
Do we think it's better to pin versions? I feel like given we have CI tests its better to let them auto upgrade and only pin if they break.